optuna/distributions.py

```diff
@@ -357,7 +333,7 @@ def __repr__(self) -> str:
         kwargs = ", ".join("{}={}".format(k, v) for k, v in sorted(self._asdict().items()))
         return "{}({})".format(self.__class__.__name__, kwargs)
 
-    def _asdict(self) -> Dict:
+    def _asdict(self) -> Dict[str, Any]:
```

Comment:
Sorry for my late comment but this seems like a somewhat unnecessary change if mypy doesn't complain. How about keeping this PR simple, and also align the type hint with `_asdict` on line 76?
```suggestion
    def _asdict(self) -> Dict:
```